templates/koa-ts/.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+	"recommendations": [
+		"esbenp.prettier-vscode",
+		"dbaeumer.vscode-eslint",
+		"bradlc.vscode-tailwindcss"
```

Comment:
```suggestion
		"esbenp.prettier-vscode",
		"dbaeumer.vscode-eslint"
```

Reply:
We won't be writing any css with this template so I think we can remove that extension! :)

Reply:
Ok, removed! 🗑️